src/main/resources/com/openshift/jenkins/plugins/pipeline/dsl/OpenShiftExec/config.jelly

```diff
@@ -0,0 +1 @@
+../../OpenShiftExec/config.jelly
```

Comment:
ah - look at that - looks like the links were reflected here ... not sure why it didn't do that for the help files.  All good - just an observation.